packages/rule-components/src/ReportDetails/ReportDetails.css

```diff
@@ -0,0 +1,12 @@
+.OCMReportDetails {
+    padding: 20px 40px;
+}
+
+.OCMReportDetailsExternalLinkIcon {
+    margin-bottom: 1px;
```

Comment:
No need to add `1px` margin.

Reply:
but it looks like there's very small margin in mock up

Reply:
Then you should use `margin-bottom: var(--pf-global--spacer--xs)`

Reply:
done, thanks